test/library/gui/widgets.Label.lua

```diff
@@ -0,0 +1,117 @@
+-- test -dhack/scripts/devel/tests -twidgets.Label
```

Comment:
There is actually a `test/library/gui/widgets.lua` file that you can add these tests to now. It was created relatively recently, so you might have to merge latest develop to see it.

Reply:
yep, wasn't there last time I checked. But maybe separate widget test files aren't bad? I imagine should every widget get tests, the single file with all the tests will be huge.

Reply:
I'm ok with separate per-widget tests